webui/react/src/pages/TrialDetails/Profiles/ProfilesEnabled.tsx

```diff
@@ -0,0 +1,56 @@
+import { Alert } from 'antd';
+import React, { useState } from 'react';
+
+import Section from 'components/Section';
+import { TrialDetails } from 'types';
+
+import SystemMetricChart from './SystemMetricChart';
+import SystemMetricFilter, { FiltersInterface } from './SystemMetricFilter';
+import TimingMetricChart from './TimingMetricChart';
+import { MetricType, useFetchMetrics } from './utils';
+
+export interface Props {
+  trial: TrialDetails;
+}
+
+const ProfilesEnabled: React.FC<Props> = ({ trial }: Props) => {
+  const [ filters, setFilters ] = useState<FiltersInterface>({});
+  const systemMetrics = useFetchMetrics(
+    trial.id,
+    MetricType.System,
+    filters.name,
+    filters.agentId,
+    filters.gpuUuid,
+  );
+
+  return (
+    <>
+
+      {systemMetrics.isEmpty && (
+        <Alert
+          message='No data available.'
+          type='warning'
+        />
+      )}
+
+      <div style={{ display: (systemMetrics.isEmpty ? 'none' : 'block') }}>
+
+        <Section
+          bodyBorder
```

Comment:
**suggestion:** make `bodyBorder` conditional depending on whether there's chart data for the section, so it would only show the yellow `Alert` box and not the surrounding section white box.

![2021-05-25 at 10 13 AM](https://user-images.githubusercontent.com/220971/119532564-3b945480-bd42-11eb-8471-b9a4a93104dc.png)

Reply:
Fixed this by moving <Section> inside <TimingMetricChart>